test/controllers/projects_controller_test.rb

```diff
@@ -152,6 +161,9 @@
       it "renders" do
         get :edit, params: {id: project.to_param}
         assert_template :edit
+        assert_select 'fieldset' do |fs|
```

Comment:
`def fields_disabled?` and then `assert fields_disabled?` + `refute fields_disabled?` for symmetry and so they never get out of sync by accident